code/src/Components/Footer.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import logo from './assests/logo.svg';
+
+const Footer = () => {
+    return (
+        <footer>
+            <img alt="Technigo logo" className="logo" src={logo}></img>
+            <div className="footer-info">
+                <p>Project 11 - Happy Thoughts</p>
+                <p>Anna Hellqvist @ Technigo Bootcamp 2020</p>
+                <div>
+                    <p>
+                        <a href="https://github.com/AnnaHellqvist">GitHub</a>
+                         -
```

Comment:
Maybe you could add an aria-label to these links, so that the screen reader reads something like "click on this link to get to Annas Github" for example 😃